### PR TITLE
Natural ceramic -- track page transitions & pupdates in segment.io/google analytics

### PR DESCRIPTION
**Pull request by judeallred**
https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=9f7963becc9341ed9af502682c6e9eef&p=a2f20aac17af492cb80cd304505bbec9

@garethx  FYI

### CODE REVIEW

File: src/presenters/includes/not-found.jsx [hunk 1]
```diff
@@ -1,12 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const cat = "https://cdn.glitch.com/us-east-1%3Acba180f4-ee65-4dfc-8dd5-f143280d3c10%2Fcat.svg";
```
Comment (judeallred) [inline src/presenters/includes/not-found.jsx]: unrelated; this change is in master/live

**Comment by judeallred** [comment]
Also added tracking for when pupdates are viewed

**Comment by judeallred** [comment]
@garethx We're now tracking a "Pupdate" event which shows whenever a user views a pupdate

File: src/presenters/overlays/new-stuff.jsx [hunk 2]
```diff
@@ -69,7 +71,10 @@ class NewStuff extends React.Component {
   renderOuter({visible, setVisible}) {
     const {children, isSignedIn, showNewStuff, newStuffReadId} = this.props;
     const dogVisible = isSignedIn && showNewStuff && (newStuffReadId < latestId);
-    const show = () => this.showNewStuff(setVisible);
+    const show = () => {
+      analytics.track("Pupdate");
```
Comment (GregWeil) [inline src/presenters/overlays/new-stuff.jsx]: Can you add an 'if analytics' check here

Comment (judeallred) [reply src/presenters/overlays/new-stuff.jsx]: sure

**Pull request merged**